common/Makefile

```diff
@@ -46,6 +47,7 @@ DEFINE_FLAGS := $(DEFINES:%=-D %)
 SHARED_FLAGS := \
 	-march=rv32im  \
 	-mabi=ilp32 \
+	-fno-math-errno \
```

Comment:
we should remove this - it got here by accident

Reply:
Removed.